src/py42/clients/detectionlists/high_risk_employee.py

```diff
@@ -6,11 +6,7 @@
 
 
 class HighRiskEmployeeClient(BaseClient):
-    """Administrator utility to manage High Risk employees.
-
-    `Support Documentation <https://support.code42.com/Administrator/Cloud/Monitoring_and_managing/Detection_list_management_APIs>`__
-
-    """
+    """A client for interacting with High Risk Employee APIs."""
```

Comment:
Employees? If it is plural for `departing employees`, it's  plural for HR as well probably